frontend/src/components/pages/cabins/Documents/Contract.tsx

```diff
@@ -16,6 +18,15 @@ const Contract: React.FC<ContractProps> = ({ chosenCabins, contactInfo, datePick
   const currentTime = new Date().toLocaleString();
   const price = calculatePrice(chosenCabins, contactInfo, datePick);
 
+  const { data } = useQuery<{ activeBookingResponsible: BookingResponsible }>(QUERY_BOOKING_RESPONSIBLE);
+  const [responsible, setResponsible] = useState<BookingResponsible>();
+
+  useEffect(() => {
+    if (data?.activeBookingResponsible) {
+      setResponsible(data.activeBookingResponsible);
+    }
+  }, [data]);
+
```

Comment:
Would this serve the same purpose?
```suggestion
  const { data } = useQuery<{ activeBookingResponsible: BookingResponsible }>(QUERY_BOOKING_RESPONSIBLE);
  const { activeBookingResponsible: responsible } = data;
```

Reply:
Results in `Property 'activeBookingResponsible' does not exist on type '{ activeBookingResponsible: BookingResponsible; } | undefined'` 🤔